sql/check_sequence_overflow.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health-sql
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+select
+    schemaname, -- schema name
+    sequencename, -- check sequence growth direction (increment or decrement)
```

Comment:
Ensure that sequence name is prefixed with schema name.
Resulted name should be like here https://github.com/mfvanek/pg-index-health-sql/blob/master/sql/non_primary_key_columns_with_serial_types.sql#L19